Dockerfile

```diff
@@ -45,9 +46,9 @@ RUN ./gradlew build pluginJar --no-daemon
 # 4. Launch a minecraft server with jdk17 and plugin
 FROM amazoncorretto:17
 # Change to the current plugin version present in build.gradle
-ENV PLUGIN_VERSION=2.9.0
+ENV PLUGIN_VERSION=3.0.0
```

Comment:
Yeah, it will be 2.10.0 because it is only a minor change (3.x.x would indicate a breaking change). 
However, I'll update that on my own, so it is fine.

Reply:
For some reason the other points where I changed the version didn't get pushed. Have fixed that now, and changed the version to 2.10.0.